modules/rosa-edit-objects.adoc

```diff
@@ -263,6 +263,20 @@ a|--kubelet-configs <kubeletconfig_name>
 |--min-replicas
 |Specifies the minimum number of compute nodes when enabling autoscaling.
 
+//OSDOCS-11160: HCP only, but need to wait on separate HCP publishing
+//ifdef::openshift-rosa-hcp[]
+|--max-surge
+a| For {hcp-title-first} clusters, the `max-surge` parameter defines the number of new nodes that can be provisioned in excess of the desired number of replicas for the machine pool, as configured using the `--replicas` parameter, or as determined by the autoscaler when autoscaling is enabled. This can be an absolute number (for example, `2`) or a percentage of the machine pool size (for example, `20%`), but must use the same unit as the `max-unavailable` parameter.
+
+The default value is `1`, meaning that the maximum number of nodes in the machine pool during an upgrade is 1 plus the desired number of replicas for the machine pool. In this situation, one excess node can be provisioned before existing nodes need to be made unavailable. The number of nodes that can be provisioned simultaneously during an upgrade is `max-surge` plus `max-unavailable`.
+
+|--max-unavailable
+a|For {hcp-title-first} clusters, the `max-unavailable` parameter defines the number of nodes that can be made unavailable in a machine pool during an upgrade, before new nodes are provisioned. This can be an absolute number (for example, `2`) or a percentage of the current replica count in the machine pool (for example, `20%`), but must use the same unit as the `max-surge` parameter.
+
+The default value is `0`, meaning that zero outdated nodes are removed before new nodes are provisioned. The valid range for this value is from `0` to the current machine pool size, or from `0%` to `100%`. The total number of nodes that can be upgraded simultaneously during an upgrade is `max-surge` plus `max-unavailable`.
```

Comment:
Suggestion: 
s/meaning that zero outdated nodes.../meaning that no outdated nodes...